### PR TITLE
Fix spelling

### PR DESCRIPTION
**Pull request by hugovk**

### CODE REVIEW

File: website/versioned_docs/version-1.0/tutorials/adding-a-module-to-your-runtime.md [hunk 1]
```diff
@@ -281,7 +281,7 @@ cargo build --release
 
 ## Next Steps
 
-We wont actually go into the details of testing the Contracts module functionality added to this chain, but if you want to try it out, you can follow our [contract tutorials](https://substrate.dev/tutorials).
+We won't actually go into the details of testing the Contracts module functionality added to this chain, but if you want to try it out, you can follow our [contract tutorials](https://substrate.dev/tutorials).
```
Comment (JoshOrndorff) [inline website/versioned_docs/version-1.0/tutorials/adding-a-module-to-your-runtime.md]: I like that you've fixed this error in the current and previous versions of the doc. Did you check thr previous versins fro the other errors you fixed in this pr?

Comment (hugovk) [reply website/versioned_docs/version-1.0/tutorials/adding-a-module-to-your-runtime.md]: Yes, the others appear only once.

**Comment by shawntabrizi** [comment]
Thanks @hugovk

**Comment by hugovk** [comment]
You're welcome!

**Pull request merged**